api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupOverDueForViralLoadReport.java

```diff
@@ -215,9 +218,15 @@ public PatientDataDefinition getObsByEndDate(Concept question, List<EncounterTyp
         return df.createPatientDataDefinition(def, converter, Parameters.combineParameters(endDate));
     }
 
+    private DataSetDefinition healthFacilityName() {
```

Comment:
@dbaluku move to SharedDataDefintion so that it is not repeated in multiple places